easyauth/server.py

```diff
@@ -274,11 +274,18 @@ def key_setup(self):
         # check if keys exist in KEY_PATH else create
         assert "KEY_NAME" in os.environ, f"missing KEY_NAME env variable"
         key_path = os.getenv("KEY_PATH", "")
+
+        if not os.path.exists(key_path):
```

Comment:
I like the idea of adding a check here for the path, but raising an exception instead of creating the path. 

The reason behind this is that with the ephemeral workers like docker / kubernetes, we would have a different private key each a new instance started instead of forcing the developer to ensure that a persistent volume is used to store the private key. For additional context, when a private key changes any existing and unexpired tokens will not work with a newly refreshed private key.